.circleci/config.yml

```diff
@@ -278,6 +278,12 @@ parameters:
   java-tag:
     type: string
     default: "17.0.4-browsers"
+  postgres-tag:
+    type: string
+    default: "13.3"
```

Comment:
This should be 13.13 to match the web service. It's beyond the scope of the issue, but would you mind trying it? If it doesn't just work, then please create a followup ticket.